### ISSUE TITLE
android port

### ISSUE THREAD

**Issue by JoseXD2** [Circuitella/Funkin-At-Freddys-Pizzeria-Demo-Public-Hotfix-Source#1]
can i work on a mobile port of this?

**Comment by Circuitella** [comment]
yeah you can make a mobile port of the demo, just ill have release handled

**Comment by lumiata342** [comment]
> yeah you can make a mobile port of the demo, just ill have release handled

I already did the port, thank you
